eclipse-collections/src/main/java/org/eclipse/collections/impl/bimap/mutable/AbstractMutableBiMap.java

```diff
@@ -283,12 +283,6 @@ public void putAll(Map<? extends K, ? extends V> map)
     }
 
     @Override
-    public V add(Pair<K, V> keyValuePair)
```

Comment:
This was removed because it was duplicate code from the default method?

Reply:
yes